integration_tests/k8s/k8sapi.go

```diff
@@ -209,13 +209,13 @@ func (c K8sClient) DeleteClusterRoleBinding(clusterrolebinding rbacv1beta1.Clust
 
 // CreateCustomResourceDefinition creates a CRD
 func (c K8sClient) CreateCustomResourceDefinition(customResourceDefinition apiextensionsV1.CustomResourceDefinition) error {
-	_, err := c.APIextClient.ApiextensionsV1().CustomResourceDefinitions().Create(&customResourceDefinition)
+	_, err := c.APIextClient.ApiextensionsV1().CustomResourceDefinitions().Create(context.Background(), &customResourceDefinition, metav1.CreateOptions{})
```

Comment:
`context.TODO()` needs to be used.

Reply:
I see `TODO` context being used at multiple places. Is there any root context we can replace these with?